applications/FemToDemApplication/python_scripts/MainCouplingFemDem.py

```diff
@@ -935,7 +935,6 @@ def CreateFEMPropertiesForDEFEContact(self):
                 max_id_properties = prop.Id
         props = KratosMultiphysics.Properties(max_id_properties + 1)
         self.created_props_id = max_id_properties + 1
-        props[KratosDEM.FRICTION] =  -0.5773502691896257
```

Comment:
Are you sure that I have to remove the FRICTION? I tried and I got an error I think :S

Reply:
Mmmmm... Let me check...

Reply:
@AlejandroCornejo, didi you tried with this branch?

Reply:
The idea is to end up removing FRICTION and substitute it by 2 variables (static and dynamic). If it's giving problems, we should check why. We can debug a minimal example if you send it to us @AlejandroCornejo .

Reply:
I've tried to remove it and the DemCouplingStructure gives an error when transfering entities

Reply:
Sorry, @AlejandroCornejo, but I cannot reproduce the error you are saying. In the master branch it should give an error, but in this branch it shouldn't. To solve that is why I change the line in the file applications/DemStructuresCouplingApplication/custom_utilities/dem_structures_coupling_utilities.h

Reply:
@AlejandroCornejo, I am properly running a simulation with the FemToDemApplication in this branch, If there is any problem, can you send me an example for debugging?

Reply:
Hi! Is it merged with the latest master? In thiscase I'll try it now

Reply:
Yes, it is.